src/components/Item/Item.component.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import 'firebase/firestore';
+import * as firebase from '../../components/Firebase/Firebase.component';
+import Checkbox from '@material-ui/core/Checkbox';
+import './Item.style.scss';
+
+const Item = props => {
+  const itemName = props.name;
+  const itemId = props.id.toString();
+  const localToken = props.localToken;
+  const over24 = props.over24;
+
+  useEffect(() => {
+    if (over24 === false) {
+      document.getElementById(itemId).setAttribute('class', 'highlight');
```

Comment:
Have we addressed the matter of establishing naming conventions for `classNames` when styling? For example, using, `.item__hightlight`?

Reply:
Thank you for pointing that out. I could change that.